pallets/connectors/src/message.rs

```diff
@@ -81,12 +144,21 @@ impl<
 	/// in other domains and MUST follow the defined standard.
 	fn call_type(&self) -> u8 {
 		match self {
-			Self::Invalid => 0,
+			Self::Invalid { .. } => 0,
```

Comment:
Out of curiosity, why is this change needed? 🤔

Reply:
It was a result of me deleting the entire match and using autofill to apply the changes 😅 I will revert it as it is not needed.